mp_locales/en-US.json

```
@@ -185,5 +185,10 @@
     "ui.apps.multiplayer.playerlist.openProfile": "Open profile",
     "ui.apps.multiplayer.playerlist.restoreVehicles": "Queue deleted vehicles",
     "ui.apps.multiplayer.playerlist.players": "Players",
-    "ui.apps.multiplayer.playerlist.error": "No players.. something is wrong"
+    "ui.apps.multiplayer.playerlist.error": "No players.. something is wrong",
+
+    "ui.options.multiplayer.enableNewChatDesign": "App Style",
+    "ui.options.multiplayer.enableNewChatDesign.tooltip": "Choose the visual style for the application's user interface.",
+    "ui.options.multiplayer.enableNewChatDesign.redesign": "Redesign",
+    "ui.options.multiplayer.enableNewChatDesign.old": "Old"
```

Could the newline here be removed and could the strings be changed to better indicate that it's about ui apps?
```suggestion
    "ui.options.multiplayer.useUiAppRedesign": "UI App Style",
    "ui.options.multiplayer.useUiAppRedesign.tooltip": "Choose the visual style for BeamMP's ui apps.",
    "ui.options.multiplayer.useUiAppRedesign.redesign": "Redesign",
    "ui.options.multiplayer.useUiAppRedesign.old": "Old"
```